tests/e2e/Event.test.ts

```diff
@@ -81,7 +81,7 @@ describe('e2e', () => {
 
       it('should return trustline updates from more than one network', async () => {
         const allEvents = await tl1.event.getAll({ type: 'TrustlineUpdate' })
-        const networks = allEvents.map(e => e.networkAddress)
+        const networks = allEvents.map(e => e.networkAddress && e.networkAddress)
```

Comment:
is this intended?

Reply:
Yes it is... looks a bit weird I know. Wanted to only filter events which have an attribute `networkAddress` because `event.getAll` now returns also token events which have an attribute `tokenAddress` instead of `networkAddress`.

Reply:
but it says `e.networkAddress && e.networkAddress`

Reply:
Does not make sense

Reply:
changed it in the last refactoring https://github.com/trustlines-network/clientlib/blob/7bbc65527548a96f65ae1c01f8ba1799c1be7fa3/tests/e2e/Event.test.ts#L86